config/external-beacon-config.go

```diff
@@ -14,6 +14,12 @@ type ExternalBeaconConfig struct {
 
 	// The URL of the Prysm gRPC endpoint (only needed if using Prysm VCs)
 	PrysmRpcUrl Parameter[string]
+
+	// Number of seconds to wait for a fast request to complete
+	FastTimeout Parameter[uint64]
+
+	// Number of seconds to wait for a slow request to complete
+	SlowTimeout Parameter[uint64]
```

Comment:
Can we add units to these field names? ie `FastTimeoutMS`

`Seconds` seems too rough of a granularity